assets/js/components/surveys/__fixtures__/survey-multi-question-conditional.json

```diff
@@ -0,0 +1,162 @@
+{
+  "survey_payload": {
+    "completion": [
+      {
+        "completion_ordinal": 1,
+        "completion_title": "First completion!",
+        "completion_text": "Because you picked answer 4 or 5 for the first question.",
```

Comment:
This is not entirely accurate because this completion also shows if you choose answer 3 on the first question.

Reply:
Added a new completion to show it when a user selects 3rd answer for the first question.